apps/hubble/src/addon/src/store/mod.rs

```diff
@@ -8,6 +8,7 @@ pub use self::user_data_store::*;
 pub use self::username_proof_store::*;
 pub use self::utils::*;
 pub use self::verification_store::*;
+pub use self::tag_store::*;
```

Comment:
mod.rs is the module definition for the `Store` module, basically, like an index.js